code/modules/shuttle/emergency.dm

```diff
@@ -70,7 +71,11 @@
 		to_chat(user, "<span class='warning'>The access level of your card is not high enough.</span>")
 		return
 
+	if (user in acted_recently)
```

Comment:
Check `cooldowns.dm`
You can easily give either the computer or the user a cooldown.

Reply:
I considered it. Timers are being used here so that the UI is updated when the cooldown goes away. I'm not sure how you'd replicate this with just a cooldown.

Reply:
Ah, you are right, for more complex behavior you'll want a custom timer.